Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Generic/EquipmentRotation/ControllerTwo.mo

```diff
@@ -9,44 +9,57 @@ block ControllerTwo
     "Continuous lead device operation"
     annotation (Evaluate=true, Dialog(enable=not lag));
 
+  parameter Boolean minLim = false
+    "Utilize minimum runtime period for a current lead device before rotation may occur"
+    annotation (Evaluate=true, Dialog(enable=not continuous));
+
   parameter Boolean initRoles[nDev] = {true, false}
     "Initial roles: true = lead, false = lag/standby"
     annotation (Evaluate=true, Dialog(tab="Advanced", group="Initiation"));
 
-  parameter Modelica.SIunits.Time stagingRuntime(
+  parameter Boolean simTimSta = true
+    "Measure rotation time from the simulation start"
+    annotation(Evaluate=true, Dialog(group="Scheduler", enable=continuous));
+
+  parameter Boolean weeInt = true
+    "Rotation is scheduled in: true = weekly intervals; false = daily intervals"
+    annotation(Evaluate=true, Dialog(group="Scheduler", enable=not simTimSta));
+
+  parameter Modelica.SIunits.Time rotationPeriod(
+    final displayUnit="h") = 1209600
+    "Rotation time period measured from simulation start"
+    annotation(Dialog(group="Scheduler", enable=simTimSta));
+
```

Comment:
Since we are updating the classes in the staging branch, it may be good time to address the issue that using `Modelica.SIunits.xxx` to specify the parameters. We would avoid specifying the parameters with `Modelica.SIunits.xxx`. For here, we could 
```
parameter Real rotationPeriod(final unit="s", final quantity="Time", displayUnit="h") = 1209600
```

Reply:
Let's write a script later to replace all these.